tools/autodeployment/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "autodeployment",
+  "version": "0.0.1",
+  "description": "A tool for automatic deployment",
+  "main": "server.js",
+  "dependencies": {
+    "dotenv": "^8.2.0",
+    "github-webhook-handler": "^1.0.0",
+    "pm2": "^4.2.3",
```

Comment:
I started to type out changes to do based on adding this, and realized it's too big.  Let's follow-up with pm2 changes later.